app/api/chat/rocket_chat.py

```diff
@@ -206,3 +206,34 @@ def get_rocket_chat_token(user: User, event: Event = None):
 
     rocket_chat = RocketChat(api_url)
     return rocket_chat.get_token(user, event)
+
+
+def rename_rocketchat_room(event: Event):
+    settings = get_settings()
+    if not (api_url := settings['rocket_chat_url']):
+        raise RocketChatException(
```

Comment:
This method call is called for every event whose name is changed, meaning it'll fail for every event which doesn't have chat enabled, which is most events because chat is disabled by default

Reply:
Still not resolved

Reply:
this is resolved now